composer.json

```
@@ -21,7 +21,7 @@
         "psr/http-server-middleware": "^1.0",
         "psr/http-server-handler": "^1.0",
         "ramsey/uuid": "^3.0 || ^4.0",
-        "doctrine/orm": "^2.7",
+        "doctrine/orm": "^2.7 || 2.8.x-dev",
```

```suggestion
        "doctrine/orm": "^2.7",
```
2.8 has been released and is now covered with `^2.7` - see https://semver.mwl.be/?package=doctrine%2Form&constraint=%5E2.7&stability=stable